README.md

```diff
@@ -25,6 +25,7 @@ There are 2 ways to use this tool:
 
 ### CLI Usage
 ```sh
+npm install -g webpack webpack-cli
```

Comment:
Ideally people should use whatever version of Webpack they are using in their project to generate the JSON output. Is this going to do that or will it defer to the globally installed version of Webpack which may be different?